pkg/odo/cli/config/set.go

```diff
@@ -49,6 +52,9 @@ type SetOptions struct {
 	configForceFlag bool
 	envArray        []string
 	now             bool
+	devfilePath     string
+	devfileObj      parser.DevfileObj
+	IsDevfile       bool
 }
 
```

Comment:
Help message also needs to updated for devfile config options.

Reply:
done